lib/svgo-node.js

```diff
@@ -13,7 +14,25 @@ exports.extendDefaultPlugins = extendDefaultPlugins;
 exports.createContentItem = createContentItem;
 
 const importConfig = async (configFile) => {
-  const config = require(configFile);
+  let config;
+  try {
+    // dynamic import expects file url instead of path and may fail
+    // when windows path is provided
+    const { default: imported } = await import(pathToFileURL(configFile));
```

Comment:
Just a comment here: ESM can be imported using dynamic import without any more work to get it working?

Reply:
With current lts yes. But we still supported node 10 until next major release so we fallback to require in case of error.